decidim-pages/config/locales/en.yml

```diff
@@ -3,7 +3,7 @@ en:
   decidim:
     features:
       pages:
-        name: Pages
+        name: Page
```

Comment:
I actually meant to add something like

```yml
name:
  one: Page
  other: Pages
```

So that from the view we can choose what to show (`t("name", count: 1)` would return `Page`, this is built-in for Rails i18n I think).